### ISSUE TITLE
Blank lines between case matches are removed

### ISSUE THREAD

**Issue by darthdeus** [chrisdone/hindent#89]
When the following code gets re-formatted using the @gibiansky style

```haskell
main = do
  case foo of
    Something -> do
      foo <- some $ function x
      foo <- some $ function x
      foo <- some $ function x
      baz

    Else -> do
      foo <- some $ function x
      foo <- some $ function x
      foo <- some $ function x
      baz

    Bar -> do
      quux
      hurr durr
```

it shrinks down to the following

```haskell
main = do
  case foo of
    Something -> do
      foo <- some $ function x
      foo <- some $ function x
      foo <- some $ function x
      baz
    Else -> do
      foo <- some $ function x
      foo <- some $ function x
      foo <- some $ function x
      baz
    Bar -> do
      quux
      hurr durr
```

basically removing all of the blank lines between the case matches. Even though it is possible to re-format the code to put a blank line above the last statement (as shown below), I don't think this is correct for larger case statements.

```haskell
main = do
  case foo of
    Something -> do
      foo <- some $ function x
      foo <- some $ function x
      foo <- some $ function x

      baz
    Else -> do
      foo <- some $ function x
      foo <- some $ function x
      foo <- some $ function x

      baz
    Bar -> do
      quux
      hurr durr
```

**Comment by gibiansky** [comment]
This is a bug and will be fixed once #90 is merged.

**Comment by gibiansky** [comment]
This should most likely be closed.
